app/lib/frontend/templates.dart

```diff
@@ -26,6 +26,7 @@ String _escapeAngleBrackets(String msg) =>
     const HtmlEscape(HtmlEscapeMode.ELEMENT).convert(msg);
 
 const HtmlEscape _HtmlEscaper = const HtmlEscape();
+const HtmlEscape _attrEscaper = const HtmlEscape(HtmlEscapeMode.ATTRIBUTE);
```

Comment:
Could be consistent with `_Html` vs. `_attr`. I know our new style guide is very confusing regarding constants, previously we had nice `CONST_VALUE` names.

Reply:
I'll do a follow-up PR to rename `_HtmlEscaper` -> `_htmlEscaper`.